packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_card.tsx

```diff
@@ -102,11 +102,11 @@ export const GuideCard = ({
       icon={<EuiIcon size="l" type={card.icon} />}
       description={
         <>
-          {progress && (
+          {progress ? (
```

Comment:
It's up to you, but you can revert this: I gave you bad advice because I need to learn React.